EIPS/eip-IMPERSONATECALL.md

```diff
@@ -0,0 +1,94 @@
+---
+eip: <undefined number>
+title: IMPERSONATECALL Opcode
+author: Sergio Demian Lerner (sergio.d.lerner@gmail.com)
+category: Core
+type: Standards Track
+status: Draft
+created: 2020-09-24
+---
+
+### Overview
+
+Add a new opcode, `IMPERSONATECALL` at `0xf6`, which is similar in idea to `CALL`, except that it impersonates a sender, i.e. the callee sees a sender different from the real caller. To prevent collisions with other deployed contract or externally owned accounts, the impersonated sender address is derived from the real caller address and a salt.
+
+### Specification
+
+`IMPERSONATECALL`: `0xf6`, takes 7 operands:
+
+- `gas`: the amount of gas the code may use in order to execute;
+- `to`: the destination address whose code is to be executed;
+- `in_offset`: the offset into memory of the input;
+- `in_size`: the size of the input in bytes;
+- `out_offset`: the offset into memory of the output;
```

Comment:
Is `out` different than `return`? If not, I think it would be best to stick with the common nomenclature.

Reply:
Ok, I'll correct it. I found that DELEGATECALL EIP-7 refers to them as out_

Reply:
I will adopt ret_offset / ret_size.